cmd/cli/install.go

```diff
@@ -293,6 +298,27 @@ func (i *installCmd) validateOptions() error {
 		return err
 	}
 
+	list, err = getControllerDeployments(i.clientSet)
+	if err != nil {
+		return err
+	}
+
+	// Check if single mesh cluster is already specified
+	for _, mesh := range list.Items {
+		singleMeshEnforced := mesh.ObjectMeta.Labels["enforceSingleMesh"] == "true"
+		name := mesh.ObjectMeta.Labels["meshName"]
+		if singleMeshEnforced {
```

Comment:
Since the single mesh feature is a cluster wide configuration, should we introduce semantics for cluster wide configurations, such as a configMap in the default namespace, as opposed to labels on deployments?

Reply:
Per offline discussions with the team, we will push this out until later when more cluster wide configurations arise.